gtpv2/message/message.go

```diff
@@ -337,6 +337,10 @@ func Parse(b []byte) (Message, error) {
 		m = &ModifyBearerRequest{}
 	case MsgTypeModifyBearerResponse:
 		m = &ModifyBearerResponse{}
+	case MsgTypeUpdateBearerRequest:
+		m = &UpdateBearerRequest{}
+	case MsgTypeUpdateBearerResponse:
+		m = &UpdateBearerResponse{}
```

Comment:
do add _test.go files with UT

Reply:
Yeah, sure. UT is not ready yet, I need some UpdateBearer examples with bytes for it.

Or I can do it with header only